web3.js/src/transaction.ts

```diff
@@ -21,6 +21,12 @@ import type {CompiledInstruction} from './message';
  */
 export type TransactionSignature = string;
 
+export enum TransactionStatus {
```

Comment:
Using a const enum means the text can get fully erased during compile.

```suggestion
export const enum TransactionStatus {
```

Reply:
Got it. Updated now.